information/licensing/Concurrent_sessions_based_licensing.md

```diff
@@ -0,0 +1,66 @@
+# Concurrent Sessions-Based Licensing
+
+Instances of @@name which are registered and paid by the subscription plans **Small Business Basic** and **Small Business Advanced** are licensed under the **Concurrent Sessions-Based** licensing.
+This type of license allows the client to run a **number of active sessions** at the same time.
+
+## Single Concurrent Session Definition
+
+A **single concurrent session** for an @@name instance is an active connection established between **a single desktop application**, or **web** or **mobile client**, on a single user's **device**, on one side, and the **servers** on which the @@name system's instance and the user database are present and operate, on the other.  
+
+This session grants access to the system and the database for this user via this single application or web client on a desktop computer, laptop, or mobile device (a smartphone, tablet, or other).
+
+### Example of a Singe Concurrent Session
+
+An example of the former would be if you start **a single instance of your desktop application** OR **a single instance of your web client**, and then use it to connect to your @@name instance covered by your subscription plan.  
+
+If no *specific conditions* apply, this would be deemed **a single active and concurrent session** between your user's application and the server.
+
+## Multiple Concurrent Sessions Definition
+
+**Multiple concurrent sessions** for an @@name instance occur when multiple active connections between **applications** and/or **web clients** are established to the **server** the @@name instance is present and is running on.  
+
+These sessions allow our clients to execute, control, manage, and oversee their business processes from multiple locations, devices, and by multiple people and automated applications simultaneously.
+
+### Example of Multiple Active and Concurrent Sessions
+
+An example of multiple concurrent sessions would be if three of your staff members start their own single instances of their applications OR web clients, and then use each of these instances to connect to your @@name instance covered by your subscription plan.  
```

Comment:
It is not of our business to talk about "your staff members" in the licensing section. State it in a different way.

Reply:
Regarding remarks: 

It is not of our business to talk about "your staff members" in the licensing section. State it in a different way. - Addressing as "your staff members" was removed. Stated in a non-client's staff members reference way.